packages/octotiger/package.py

```diff
@@ -252,7 +252,7 @@ def cmake_args(self):
                                    or spec.satisfies("griddim=16")):
             raise SpackError("Octo-Tiger tests only work with griddim=8 and griddim=16. "
                              "Disable tests or change griddim!")
-        if not self.run_tests or (spec.satisfies("%arm") or spec.satisfies("%clang") or
+        if not self.run_tests or (spec.satisfies("%arm") or spec.satisfies("%clang") or spec.target == "neoverse_v2" or
```

Comment:
I don't think adding the  neoverse_v2 condition by itself is  actually sufficient. A grace grace machine shows me neoverse_v2, but a grace hopper one shows neoverse_n1.

Besides, the sheer number of cases where we disable the Blast tests suggests that we should inverse the condition and maybe only activate the Blast test when we build with gcc/nvcc on a x86 platform (or boost multiprecision is activated).

I will try to get a Grace Hopper build running first before deciding on that though. If that's the only fix required I am fine with just adding another neoverse_n1 condition for now, if not we might as well do it properly as it seems we encounter this quad math issue on every new platform we use.

Reply:
Yes, maybe inverting the condition would be better.

Reply:
For now, I pushed a commit that adds the exception to the #5 PR (as it contains the other changes required for both the Grace and Intel GPU machines that I use. I think it makes sense to push those together into one PR).

Reply:
I would do something like:
```suggestion
        if not self.run_tests or (spec.satisfies("%arm") or spec.satisfies("%clang") or re.search(r"neoverse",spec.target) or
```